labs/src/main/scala/org/scalalabs/basic/lab03/PatternMatchingExercise.scala

```diff
@@ -55,7 +76,10 @@ object PatternMatchingExercise {
    *    otherwise return `None`
    */
   def older(p: Person): Option[String] = {
-    error("fix me")
+    p.age match {
+      case x if x <= 30 => None
+      case _ => Option(p.name)
+    }
```

Comment:
I think in this case I would write to avoid <= 30 =>, the visual intuition (subconscious pattern matching?) is confusing!